velox/exec/SimpleAggregateAdapter.h

```diff
@@ -145,6 +145,18 @@ class SimpleAggregateAdapter : public Aggregate {
   struct support_to_intermediate<T, std::void_t<decltype(&T::toIntermediate)>>
       : std::true_type {};
 
+  // Whether the accumulator requires aligned access. If it is defined,
+  // SimpleAggregateAdapter::accumulatorAlignmentSize() returns
+  // alignof(typename FUNC::AccumulatorType).
+  // Otherwise, SimpleAggregateAdapter::accumulatorAlignmentSize() returns
+  // Aggregate::accumulatorAlignmentSize(), with a default value of 1.
+  template <typename T, typename = void>
+  struct aligned_accumulator : std::false_type {};
```

Comment:
Great! Thank you for helping enhance the aggregation function interface! Could you also update the documentation about this new flag? (https://facebookincubator.github.io/velox/develop/aggregate-functions.html, source code is in velox/docs/develop/aggregate-functions.rst.)